packages/at_secondary_server/pubspec.yaml

```diff
@@ -35,6 +35,14 @@ dependencies:
   yaml: 3.1.2
   logging: 1.2.0
 
+dependency_overrides:
+  at_commons:
```

Comment:
Before merging to trunk, please replace the dependency overrides with published version.

Reply:
replaced the override with v3.0.56 in [a8fabe0](https://github.com/atsign-foundation/at_server/pull/1562/commits/a8fabe018a09e5faeae6d51bc5614b721dd3881f)